L1Trigger/L1TMuonBarrel/src/L1MuBMTQualPatternLut.cc

```diff
@@ -215,8 +218,24 @@ const vector<short>& L1MuBMTQualPatternLut::getQualifiedPatterns(int sp, int adr
 
   LUT::const_iterator it = m_lut.find(make_pair(sp,adr));
   if ( it == m_lut.end() ) {
-    cerr << "Error: L1MuDTQualPatternLut: no pattern list found for address " << adr << endl;
+    edm::LogError ("Not found") << "Error: L1MuDTQualPatternLut: no pattern list found for address " << adr << endl;
   }
   return (*it).second.second;
 
 }
+
+
+int L1MuBMTQualPatternLut::getIgnoredLines(L1TriggerLutFile file) const{
+    if ( file.open() != 0 ) return -1;
+  int skip=0;
+  while ( file.good() ) {
+     string str=file.readString();
+     if (str.find("#")==0) skip+=1;
+     // cout<<"here "<<str<<" found "<<str.find("#")<<endl;
+        if ( !file.good() ) { file.close(); break; }
+    
+  }
+  // cout<<"coarse lines to skip3 "<<skip<<endl;
+  file.close();
+  return skip;
+}
```

Comment:
@gkaratha can you reuse the function already defined in L1MuBMTEtaPatternLut? They seem to be the same.